rhelai-validation/inventory

```diff
@@ -1,2 +1,5 @@
+[control_node]
+localhost
+
 [target_nodes]
-rhelai0 ansible_host=192.168.122.222 ansible_user=cloud-user ansible_ssh_private_key_file=~/.ssh/rhel-ai.pem ansible_ssh_common_args='-o StrictHostKeyChecking=accept-new'
```

Comment:
@lpiwowar You'll recognize this one. :) Now that we're potentially destroying/creating the VM on each execution, we do need to step down to this.

Reply:
Ah, it is back:). I'm do not see through this entirely. Won't it be an issue when we will be `ssh` for the first time to the VM that gets created using `deploy_vm.yaml`? During which step are we copying the key of the VM to `known_hosts`?

Reply:
At the moment, we are never copying it into known_hosts, that's why it's being set to `no`.